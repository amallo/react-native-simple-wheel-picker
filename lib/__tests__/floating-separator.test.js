import React from 'react'
import TestRenderer from 'react-test-renderer';
import FloatingSeparator from '../floating-separator';


it('renders snapshot', () => {
    const wrapper = TestRenderer.create(
        <FloatingSeparator
            testID={'bottomSeparator'}
            height={40}
            position={200}
            backgroundColor={'red'}
            marginHorizontal={40} />);
    expect(wrapper.toJSON()).toMatchSnapshot();
});

it('renders a separator', () => {
    const wrapper = TestRenderer.create(
        <FloatingSeparator
            height={40}
            position={200}
            backgroundColor={'red'}
            marginHorizontal={40} />);

    const separator = wrapper.root.findByProps({ testID: 'floatingSeparatorView' })
    expect(separator.props.style).toContainEqual({
        height: 40,
        backgroundColor: 'red',
        top: 200,
        left: 40,
        right: 40
    })
})