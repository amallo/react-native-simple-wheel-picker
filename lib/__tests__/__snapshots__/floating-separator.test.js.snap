// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders snapshot 1`] = `
<View
  style={
    Array [
      Object {
        "position": "absolute",
      },
      Object {
        "backgroundColor": "red",
        "height": 40,
        "left": 40,
        "right": 40,
        "top": 200,
      },
    ]
  }
  testID="floatingSeparatorView"
/>
`;
